backend/src/nodes/properties/inputs/generic_inputs.py

```diff
@@ -223,12 +223,32 @@ def VideoTypeDropdown() -> DropDownInput:
         label="Video Type",
         options=[
             {"option": "MP4", "value": "mp4"},
+            {"option": "MKV", "value": "mkv"},
+            {"option": "Webm", "value": "webm"},
             {"option": "AVI", "value": "avi"},
             {"option": "None", "value": "none"},
         ],
     )
 
 
+def VideoPresetDropdown() -> DropDownInput:
+    """Video Type option dropdown"""
+    return DropDownInput(
+        input_type="VideoPreset",
+        label="Preset",
+        options=[
+            {"option": "ultrafast", "value": "ultrafast"},
+            {"option": "superfast", "value": "superfast"},
+            {"option": "veryfast", "value": "veryfast"},
+            {"option": "fast", "value": "fast"},
+            {"option": "medium", "value": "medium"},
+            {"option": "slow", "value": "slow"},
+            {"option": "slower", "value": "slower"},
+            {"option": "veryslow", "value": "veryslow"},
```

Comment:
Why not use nice labels here?

Reply:
Because these are standard ffmpeg terms. Unless you mean capitalize them or something

Reply:
That's alright then.